src/components/QuestionBox.tsx

```diff
@@ -0,0 +1,50 @@
+import { useState } from "react";
+
+const questionBox = ({
+  question,
+  answer,
+}: {
+  question: string;
+  answer: string;
+}) => {
+  const [showed, setShowed] = useState(false);
+
+  return (
+    <div className="mb-6 flex flex-col rounded-3xl bg-white shadow-lg">
+      <div
+        className="my-3 flex flex-row items-center justify-between rounded-3xl px-8 py-4 "
+        onClick={() => setShowed(!showed)}
+      >
+        <div className="flex items-center justify-between">
+          <div className="flex items-center">
+            <img
+              className="mr-3 h-6 w-6"
+              src="../../public/question_icon.svg"
```

Comment:
`img` ให้ใช้เป็น import แทน ดูได้ที่ [react-docs](https://create-react-app.dev/docs/adding-images-fonts-and-files/)

คือ เก็บไฟล์ไว้ใน folder `src/assets` จะสร้างโฟลเดอร์ใหม่หรือยังไงก็ตามแต่ แล้วเวลาเรียกใช้ก็ import มา